### PR TITLE
806 frontend refine events component according to figma

### PR DESCRIPTION
**Pull request by zlrkw11**
Image cover (URL prop) display on top of the Events Card
Events Card Storybook group set up
Adjustments & Debugging

### CODE REVIEW

**Comment by zlrkw11** [comment]
image

**Comment by zlrkw11** [comment]
image

**Comment by zlrkw11** [comment]
- image added
- props adjusted 
- storybook check for stacked components

File: client/src/assets/images/test.png
Comment (bcho892) [inline client/src/assets/images/test.png:1]: delete this and use an url instead

**Review by bcho892 — CHANGES_REQUESTED** [review]
follow [proper guidelines](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/best-practices-for-pull-requests#provide-context-and-guidance) for a PR description so future contributors can determine the purpose

**Review by bcho892 — APPROVED** [review]

**Pull request merged**